applications/doodle/src/tf_exe.erl

```diff
@@ -374,8 +374,7 @@ tf_module_task(TFModule, Data, Im, AMQPConsumer, AMQPChannel) ->
 
```

Comment:
Unrelated to here but:
tf_exe:287 calls `stop/2` with `'finish'` as the second arg. `stop/2` spec has `kz_term:api_ne_binary()`. Probably just use `any()` since it is passed as the shutdown arg in `handle_cast/2`

Reply:
yep